lib/fxpay/settings.js

```diff
@@ -80,12 +83,18 @@
         exports[def] = defaultSettings[def];
       }
     }
-    for (var k in newSettings) {
-      if (typeof exports[k] === 'undefined') {
-        exports.log.error('configure() received an unknown setting:', k);
+    for (var param in newSettings) {
```

Comment:
Well, I looked up Object.keys() and it's only IE9 and greater so it would be better to avoid it if possible. Also, in this case I actually think prototype chains might be useful if someone wanted to use inheritance for their settings. But, meh, probably not.